presto-main/src/main/java/com/facebook/presto/server/RemoteTaskStats.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.server;
+
+import io.airlift.stats.DistributionStat;
+import org.weakref.jmx.Managed;
+import org.weakref.jmx.Nested;
+
+public class RemoteTaskStats
+{
+    private final DistributionStat updateRoundTripMillis = new DistributionStat();
+    private final DistributionStat infoRoundTripMillis = new DistributionStat();
+    private final DistributionStat responseSizeBytes = new DistributionStat();
```

Comment:
The response for both update and info are the `TaskInfo` object. So I have only one. In the new code, I will have two stats, that will have a `TaskInfo` and `TaskStatus`.